packages/components/src/components/combobox/shadow.tsx

```diff
@@ -39,7 +39,7 @@ import { translate } from '../../i18n';
 })
 export class KolCombobox implements ComboboxAPI {
 	@Element() private readonly host?: HTMLKolComboboxElement;
-	private ref?: HTMLSelectElement;
+	private ref?: HTMLInputElement;
```

Comment:
```suggestion
	private refInput?: HTMLInputElement;
```

(Ich habe in einem anderen Branch schon angefangen, das zu vereinheitlichen, ist nur noch nicht gemerged. Dann weiß man auch, welcher `ref` für welches Element steht.)